R/plotDMN.R

```diff
@@ -55,7 +55,7 @@ setMethod("plotDMNFit", signature = c(x = "SummarizedExperiment"),
             dmn <- metadata(x)[[name]]$dmm
         } else {
             .Deprecated(old="getDMN", new="cluster", 
-                    "Now runDMN and calculateDMN are deprecated. Use cluster with DMMParam parameter and full parameter set as true instead.")
+                    "Now runDMN and cluster are deprecated. Use cluster with DMMParam parameter and full parameter set as true instead.")
```

Comment:
Here is error

Reply:
Also add msg="Now runDMN...." --> the deprecation message is not printed correctly (I have noticed that in different functions)